variables.tf

```diff
@@ -74,6 +69,23 @@ variable "monitoring_enabled" {
   description = "Enable monitoring for the Auto Scaling Group"
 }
 
+variable "tailscale_api_token" {
+  type        = string
+  description = "Set Tailscale API access token here"
+}
+
+variable "tailscale_key_expiry" {
+  type        = string
+  default     = 7776000
+  description = "The expiry of the key in seconds. Defaults to 7776000 (90 days)"
+}
+
+variable "tailscale_tags" {
+  type        = list(string)
+  default     = ["tag:server"]
```

Comment:
If you use concat as suggested earlier, here it should become an empty list IMO.